### PR TITLE
[neo2] Raise priority of Ping/Pong

### PR DESCRIPTION
**Pull request by Ashuaidehao**
Raise priority of Ping/Pong to ensure the reliability of syncing blocks  process，Close https://github.com/neo-project/neo/issues/1806.

### CODE REVIEW

File: neo/Network/P2P/TaskManager.cs [hunk 1]
```diff
@@ -217,8 +218,15 @@ public static Props Props(NeoSystem system)
             return Akka.Actor.Props.Create(() => new TaskManager(system)).WithMailbox("task-manager-mailbox");
         }
 
+        private readonly ConcurrentDictionary<ActorPath, DateTime> _expiredTimes = new ConcurrentDictionary<ActorPath, DateTime>();
```
Comment (Tommo-L) [inline neo/Network/P2P/TaskManager.cs:221]: `Dictionary<ActorPath, DateTime>` is enough?

Comment (Tommo-L) [reply neo/Network/P2P/TaskManager.cs:221]: Better move it to L43.

Comment (Ashuaidehao) [reply neo/Network/P2P/TaskManager.cs:221]: `Dictionary<ActorPath, DateTime>` is not thread-safe, may cause insert error in concurrent scenarios.

**Comment by Ashuaidehao** [comment]
Maybe add a `Timer` ?

**Comment by shargon** [comment]
yes, a task it's good, something to prevent an `outOfMemory`.

**Comment by shargon** [comment]
@Ashuaidehao please take a look to my solution for neo3 without a new collection. https://github.com/neo-project/neo/pull/1829 I think that we can do the same here

**Comment by shargon** [comment]
Sorry for close, miss click

**Comment by superboyiii** [comment]
@shargon Test PASS, is able to merge. (from BSN side, the issue provider, they work well after this PR till now)

**Pull request merged**